### ISSUE TITLE
Validate `placekey-py` against newest version of H3 Library

### ISSUE THREAD

**Issue by jarredSafegraph** [Placekey/placekey-py#41]
Users have reported latest version of `placekey-py` are not working with latest versions of H3.

**Comment by eric-leap** [comment]
@jarredSafegraph Indeed. I temporarily solved this by specifying `pip install h3<4.0.0` before I install `placekey`. But it seems like `placekey` should be updated to reflect the updated version of h3.

**Comment by bohrarohan** [comment]
Thanks @eric-leap 
We too were able to resolve issues(_short-term_solution_) using your suggestion, and installing h3 v2.7.7 🚀

**Comment by jarredSafegraph** [comment]
Hey,

Sorry for the delay on this one. In taking a look at the changes to the H3 library and wanting to ensure we maintain functionality in our library, we have decided to lock on h3<4,>=3.6 for the time being. If you are working with the newest version of H3, do let me know, and I can see about making the necessary changes. 

```
Collecting h3<4,>=3.6.1 (from placekey)
  Downloading h3-3.7.7-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (4.9 kB)
```

**Comment by medhatphq** [comment]
Hi @jarredSafegraph I also am having problems getting placekey-py to work nicely with h3-py, specifically I need a version of h3-py that allows me to convert H3 cells to GeoJSON/Polygons and AFAIK this was only possible in h3-py since version `4` which is incompatible with all versions of placekey-py.

**Comment by medhatgayed** [comment]
Hi @jarredSafegraph I also am having problems getting placekey-py to work nicely with h3-py, specifically I need a version of h3-py that allows me to convert H3 cells to GeoJSON/Polygons and AFAIK this was only possible in h3-py since version 4 which is incompatible with all versions of placekey-py.

**Comment by jarredSafegraph** [comment]
@medhatgayed 
That is correct for the time being. We are weighing the path forward at the moment: either splitting API and Spatial functionality from API or something to that end. Would you mind sharing more about your use case for the H3 / spatial side? If you would rather share with me directly, you can email me at jparrett@safegraph.com.
